cloud/workspace/workspace_token.go

```diff
@@ -405,8 +431,6 @@ func getWorkspaceToken(id, name, workspace, message string, tokens []astrocore.A
 				return astrocore.ApiToken{}, err
 			}
 		}
-	case name != "" && id != "":
```

Comment:
Do we feel we need this error? Currently if an id is found it queries by id to grab the token and thus never gets to this block. If we feel we do need this error I can move it higher (before the api call)

Reply:
don't think we need this if we are going with using ID directly to check for token if passed in the command